test/manifests/demuxed-two.m3u8

```diff
@@ -0,0 +1,6 @@
+#EXTM3U
+#EXT-X-VERSION:4
+#EXT-X-MEDIA:TYPE=AUDIO,GROUP-ID="audio",NAME="en",DEFAULT=YES,AUTOSELECT=YES,LANGUAGE="en",URI="media.m3u8"
+#EXT-X-STREAM-INF:PROGRAM-ID=1,BANDWIDTH=564300,CODECS="mp4a.40.2,avc1.420015",AUDIO="audio"
+media.m3u8
```

Comment:
The URI is the same for both alt audio and muxed.

Reply:
yeah for my use case I only needed the m3u8 to be an existing manifest, in `demuxed` it isn't. That is why I added `demuxed-two`, as I don't want to break or change anything else.

Reply:
If you want I can make it so that they use different manifests, but they won't actually be used for this test and generating valid manifests might take a bit more time.

Reply:
I'm not sure I understand. I was just pointing out that they're the same because I'm not sure this would be a valid manifest. I don't see something specific in the HLS spec that goes against it, but it would be strange to see in the real world.